src/test/java/com/solutions/shop/repository/ProductRepositoryTest.java

```diff
@@ -0,0 +1,24 @@
+package com.solutions.shop.repository;
+
+import com.solutions.shop.model.Product;
+import java.math.BigDecimal;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+@DataJpaTest
+public class ProductRepositoryTest {
```

Comment:
extend tests with full CRUD

Reply:
Done. I'm not sure only, if I correctly Unit-test for "delete" made.